controllers/druid/handler.go

```diff
@@ -1360,10 +1360,10 @@ func makeDeloymentListEmptyObj() *appsv1.DeploymentList {
 	}
 }
 
-func makePodDisruptionBudgetListEmptyObj() *v1beta1.PodDisruptionBudgetList {
-	return &v1beta1.PodDisruptionBudgetList{
+func makePodDisruptionBudgetListEmptyObj() *policyv1.PodDisruptionBudgetList {
+	return &policyv1.PodDisruptionBudgetList{
 		TypeMeta: metav1.TypeMeta{
-			APIVersion: "policy/v1beta1",
```

Comment:
Any chance we can make this a constant somewhere and use that constant here and below?